package.json

```diff
@@ -63,10 +63,18 @@
     },
     "patchedDependencies": {
       "monaco-editor@0.52.2": "patches/monaco-editor@0.52.2.patch"
-    }
+    },
+    "onlyBuiltDependencies": [
```

Comment:
TIL it can be in `package.json`

https://pnpm.io/9.x/package_json#pnpmonlybuiltdependencies